packages/react-native-web/src/exports/View/index.js

```diff
@@ -69,13 +71,16 @@ const View: React.AbstractComponent<ViewProps, HTMLElement & PlatformMethods> =
     } = props;
 
     if (process.env.NODE_ENV !== 'production') {
-      React.Children.toArray(props.children).forEach((item) => {
-        if (typeof item === 'string') {
-          console.error(
-            `Unexpected text node: ${item}. A text node cannot be a child of a <View>.`
-          );
-        }
-      });
+      // Skip parsing RSC objects as this will prevent errors from surfacing in the renderer.
+      if (!props.children || !('$$typeof' in props.children)) {
```

Comment:
This fails when the child is a string

Reply:
I think the issue may have been coming from a mismatched version of `react-is` that I had. Updating everything seems to fix the issue, I'll remove the statement.